README.md

```diff
@@ -24,10 +24,11 @@ When compiling a version from source, you are going to need to install [all requ
 
 `node-build` already has a [handful of settings](https://github.com/nodenv/node-build#custom-build-configuration), in additional to that `asdf-nodejs` has a few extra configuration variables:
 
+- `ASDF_NODEJS_VERBOSE_INSTALL`: Enables verbose output for downloading and building. Any value different from empty is treated as enabled.
+- `ASDF_NODEJS_FORCE_COMPILE`: Forces compilation from source instead of preferring pre-compiled binaries
```

Comment:
It might be good to specify in the docs that this variable being set isn't enough to trigger compilation. It needs to be set to a non-empty string. 

Or it might be good to make it so it only has to be set to trigger compilation (https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash)